src/database/entities/market.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity({ name: 'market_data' })
```

Comment:
This shouldn't be flagged. It's not an entity, it's a JSONB structure.

Reply:
absolutely, my bad.